src/test/git-elegant-clear-local.bats

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bats -ex
+
+load commons
+
+setup() {
+    fake-pass git "branch -lvv | grep gone | awk {'print \$1'}"
```

Comment:
@ivan-podorozhnyi `git branch -d` is not used in your tests. Please return at least one branch and make sure it will be deleted.

Reply:
@extsoft ok